apps/handler/.env.example

```diff
@@ -15,6 +15,7 @@ DATABASE_MAX_OPEN_CONNECTIONS=5
 # sets the maximum number of connections in the idle connection pool
 DATABASE_MAX_IDLE_CONNECTIONS=5
 
+DAO_CONTRACTS_INSTALL_BLOCKS=1875320
```

Comment:
Let's pass the starting block and sync everything we missed. It will provide an easier interface and it will be a safer approach in general. Let's call the variable `SYNC_SINCE_BLOCK_HEIGHT`